CommonLibF4/include/RE/Bethesda/BSExtraData.h

```diff
@@ -619,11 +619,7 @@ namespace RE
 			return !((stl::to_underlying(a_type) - 11) & ~0x22u) && a_type != EXTRA_DATA_TYPE::kLeveledCreature;
 		}
 
-		void CreateFlags()
-		{
-			_flags = calloc<std::uint8_t>(N);
-			std::memset(_flags, 0, N);
-		}
+		void CreateFlags() { _flags = calloc<std::uint8_t>(N); }
```

Comment:
Can you convert this to a revert commit?

Reply:
Sure, this is the revert commit: https://github.com/Ryan-rsm-McKenzie/CommonLibF4/pull/10/commits/f7652b24f68516d204adacae6597e10091adbf6f